packages/data-edge/.husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
These husky files should be at the repo level not package. We already have one `commit-msg` which is identical to this one so we can delete it.